torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -974,7 +975,8 @@ def convert(self,
                         modules,
                         quantized_graph,
                         node_name_to_scope,
-                        is_input=False)
+                        is_input=False,
+                        output_prefix="")
```

Comment:
I feel we might need to break BC soon since we will not be using this code

Reply:
Ok, we can do that in a separate PR